### PR TITLE
Feat/arrs ovid streaming service

### PR DESCRIPTION
**Pull request by FonduemangVI**
# Pull Request

## Purpose
Add OViD Streaming services to the guides

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by FonduemangVI** [comment]
@TRaSH- do you want me to add this to the sonarr-streaming-services.md file? I haven't currently as all the entries in that file have scores

File: docs/Sonarr/sonarr-collection-of-custom-formats.md [hunk 1]
```diff
@@ -1774,6 +1775,24 @@ I also made 3 guides related to this one.
 
 ------
 
+### OViD
+
+<sub>OViD</sub>
+
+??? question "CRAVE - [CLICK TO EXPAND]"
```
Comment (yammes08) [inline docs/Sonarr/sonarr-collection-of-custom-formats.md:1782]: Wrong name here

File: docs/Radarr/Radarr-collection-of-custom-formats.md [hunk 2]
```diff
@@ -1919,6 +1920,24 @@ I also made 3 guides related to this one.
 
 ------
 
+### OViD
+
+<sub>OViD</sub>
+
+??? question "CRAVE - [CLICK TO EXPAND]"
```
Comment (yammes08) [inline docs/Radarr/Radarr-collection-of-custom-formats.md:1927]: Wrong name here

File: docs/json/sonarr/cf/ovid.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "trash_id": "fb1a91cdc0f26f7ca0696e0e95274645",
```
Comment (yammes08) [inline docs/json/sonarr/cf/ovid.json:2]: Sonarr streaming services have a default score of 50 which needs adding here.

**Review by yammes08 — CHANGES_REQUESTED** [review]
Sonarr streaming services have a default score of 50, and would need adding to the relevant guide tables.

**Comment by yammes08** [comment]
> @TRaSH- do you want me to add this to the sonarr-streaming-services.md file? I haven't currently as all the entries in that file have scores

Mentioned in the review, but this one will need a score of 50 (the default score for Sonarr). It can then go in the guide tables.

**Comment by FonduemangVI** [comment]
> > @TRaSH- do you want me to add this to the sonarr-streaming-services.md file? I haven't currently as all the entries in that file have scores
> 
> Mentioned in the review, but this one will need a score of 50 (the default score for Sonarr). It can then go in the guide tables.

Ah whoops I didn't realize we had switched to using a 50 default score

File: docs/Radarr/Radarr-collection-of-custom-formats.md (hunk 2 above)
Comment (FonduemangVI) [reply docs/Radarr/Radarr-collection-of-custom-formats.md:1927]: Fixed

File: docs/Sonarr/sonarr-collection-of-custom-formats.md (hunk 1 above)
Comment (FonduemangVI) [reply docs/Sonarr/sonarr-collection-of-custom-formats.md:1782]: Fixed

File: docs/json/sonarr/cf/ovid.json (hunk 3 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/ovid.json:2]: Added

**Comment by yammes08** [comment]
> > > @TRaSH- do you want me to add this to the sonarr-streaming-services.md file? I haven't currently as all the entries in that file have scores
> > 
> > 
> > Mentioned in the review, but this one will need a score of 50 (the default score for Sonarr). It can then go in the guide tables.
> 
> Ah whoops I didn't realize we had switched to using a 50 default score

It's not reallyt documented anywhere so it's not really your fault 😄 - just gonna review again.

File: includes/cf/sonarr-streaming-services.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ??? abstract "Streaming Services - [CLICK TO EXPAND]"
     | Custom Format                                                                           | Score                                     | Trash ID                               |
-    | --------------------------------------------------------------------------------------- | ----------------------------------------- | -------------------------------------- |
+    | :-------------------------------------------------------------------------------------- | :---------------------------------------- | :------------------------------------- |
```
Comment (yammes08) [inline includes/cf/sonarr-streaming-services.md:3]: What does the `:` do at the top of a table?

**Review by yammes08 — CHANGES_REQUESTED** [review]
OViD also needs adding to the Sonarr UHD streaming services table (includes/cf) - I think we always add them in both places regardless of whether they're known for UHD releases or not.

File: includes/cf/sonarr-streaming-services.md (hunk 4 above)
Comment (FonduemangVI) [reply includes/cf/sonarr-streaming-services.md:3]: It's for alignment
"Alignment
You can align text in the columns to the left, right, or center by adding a colon (:) to the left, right, or on both side of the hyphens within the header row."

Not specifically needed to vscode addon that aligns tables chucked it in. I'll just remove it

Comment (yammes08) [reply includes/cf/sonarr-streaming-services.md]: Thanks! TIL.

**Review by yammes08 — APPROVED** [review]
Niiiiiiiiiiiiiice 😎

**Comment by FonduemangVI** [comment]
Good catch on the UHD table

**Pull request merged**